src/actions/actions.js

```diff
@@ -1,4 +1,4 @@
-import { FETCH_MONSTERS, FETCH_ONE_MONSTER, SEARCH_MONSTER_NAME, GET_MONSTERS_BY_CR, ADD_TO_ENCOUNTER } from './types'
+import { FETCH_MONSTERS, FETCH_ONE_MONSTER, SEARCH_MONSTER_NAME, GET_MONSTERS_BY_CR, ADD_TO_ENCOUNTER, REMOVE_FROM_ENCOUNTER } from './types'
 
```

Comment:
I'm getting to think that it's actually pretty cool to have all the actions in one file like this.